packages/theme/pages/Checkout/Shipping.vue

```diff
@@ -373,7 +373,7 @@ export default {
     };
 
     const populateSelectedAddressId = () => {
-      if (checkoutShippingAddress.value && savedAddresses.value?.addresses) {
+      if (checkoutShippingAddress.value && savedAddresses.value?.addresses && savedAddresses.value?.addresses?.length) {
```

Comment:
There was a case where `savedAddresses.value.addresses` was not an array which caused an error. I'm not sure if that's a normal behavior. If yes then we might want to add some better check to make sure that it's an array before we run the `find` function on it.